test/serving.ts

```diff
@@ -0,0 +1,44 @@
+import { join, resolve } from 'path'
+import { fileURLToPath } from 'url'
+
+import test from 'ava'
+import fetch from 'node-fetch'
+
+import { serve } from '../src/index.js'
+
+const url = new URL(import.meta.url)
+const dirname = fileURLToPath(url)
+const fixturesDir = resolve(dirname, '..', 'fixtures')
+
+test.serial('bundler serving functionality', async (t) => {
+  const PORT_THATS_HOPEFULLY_FREE = 5832
```

Comment:
Can we call this `port`?

Also, should we use something like https://www.npmjs.com/package/get-port rather than hardcoding a port and hoping that it's free?

Reply:
good call, yeah :D done in https://github.com/netlify/edge-bundler/pull/116/commits/709cc70610a8223550d6f2e915d23e5eecfcbb7e